Makefile.am

```diff
@@ -1,3 +1,7 @@
+ACLOCAL_AMFLAGS = -I m4
+#AM_CPPFLAGS = -I$(top_builddir)/gnulib -I$(top_srcdir)/gnulib
+DEFAULT_INCLUDES = -I$(top_srcdir)/gnulib
+SUBDIRS = gnulib
 
```

Comment:
I don't think libgnu.a is our prodcuts though we will put libgnu.a in our source tree.
To represent libgnu.a is a separated product, I would like to introduce a variable like:

```
LIBGNU_LIBS = gnulib/libgnu.a $(LIBPTHREAD)
```

So <A>...

Reply:
I've fixed, but I chose a name `GNULIB_LIBS`.